File: app/src/main/java/knf/kuma/tv/AnimeRow.java
package knf.kuma.tv;

import android.support.v17.leanback.widget.ArrayObjectAdapter;

public class AnimeRow {

    private int page;
    private int id;
    private ArrayObjectAdapter adapter;
    private String title;

    public AnimeRow() {
    }

    public int getPage() {
        return page;
    }

    public AnimeRow setPage(int page) {
        this.page = page;
        return this;
    }

    public int getId() {
        return id;
    }

    public AnimeRow setId(int id) {
        this.id = id;
        return this;
    }

    public ArrayObjectAdapter getAdapter() {
        return adapter;
    }

    public AnimeRow setAdapter(ArrayObjectAdapter adapter) {
        this.adapter = adapter;
        return this;
    }

    public String getTitle() {
        return title;
    }

    public AnimeRow setTitle(String title) {
        this.title = title;
        return this;
    }
}

File: app/src/main/java/knf/kuma/tv/AnimeRow.kt
package knf.kuma.tv

import androidx.leanback.widget.ArrayObjectAdapter

class AnimeRow {

    internal var page: Int = 0
    internal var id: Int = 0
    internal var adapter: ArrayObjectAdapter? = null
    internal var title: String? = null

    fun getPage(): Int {
        return page
    }

    fun setPage(page: Int): AnimeRow {
        this.page = page
        return this
    }

    fun getId(): Int {
        return id
    }

    fun setId(id: Int): AnimeRow {
        this.id = id
        return this
    }

    fun getAdapter(): ArrayObjectAdapter? {
        return adapter
    }

    fun setAdapter(adapter: ArrayObjectAdapter): AnimeRow {
        this.adapter = adapter
        return this
    }

    fun getTitle(): String? {
        return title
    }

    fun setTitle(title: String): AnimeRow {
        this.title = title
        return this
    }
}
